First case.R

```diff
@@ -0,0 +1,92 @@
+library(tidyverse)
+library(lubridate)
+
+#Calculates start date for each country
+
+#col_num <- apply(cases[,5:ncol(cases)], 1, function(x){first_case = min(which(x>0))})
+
+#first_case <- names(cases[,5:ncol(cases)])[col_num]
+
+#cases <- cbind(first_case, cases)
+
+
+# reading in files
+
+cases = read.csv(".\\intermediate_data\\jhu_cases_01.26.2021.csv", check.names = FALSE)
+
+deaths = read.csv(".\\intermediate_data\\jhu_deaths_01.26.2021.csv", check.names = FALSE)
+
+population = read.csv(".\\intermediate_data\\world_bank_population.csv", check.names = FALSE)
+
+indicators <- read.csv(".\\intermediate_data\\ghsi_summary.csv", check.names = FALSE)
+
+
+#pivoting dates into rows and grouping by country code and date
+
+cases_country_date <- cases[7:376]
```

Comment:
I would not use indices to subset here, in case the size of our dataset ever changes. What are you trying to do here?

Reply:
@kathrynline I fixed this so it now looks for the last column of the dataset as the last date. I'm not sure how to dynamically find the first date without hardcoding anything. Let me know if you have any ideas, I just committed my updated code.

Reply:
I think using a hardcoded value for the first column is fine, it is just the last column that should be dynamic in case the number of columns increases.